trainTestReinforcementAlgorithm.py

```diff
@@ -172,21 +199,21 @@ def main():
     aparser.add_argument("--visualize_testing", type=bool, default=True)
     aparser.add_argument("--visualization_fps", type=int, default=30)
 
-    # FIXME currently we are not able to pass an argument for selecting the reward function as a command line arg. 
-    # We should figure this out, but until then, change the reward function on line 181.
-    # aparser.add_argument("--reward_function", type=Callable[..., float], default=RewardFuncs.basic_reward_func, help="function to determine how a snake agent is rewarded/punished for certain actions during training. Available functions can be found in snakeRewardFuncs.py")
-    
     aparser.add_argument("--print_analysis", type=bool, default=True, help="bool to determine whether or not analysis of test scores is done")    
     
     aparser.add_argument("--save_model", type=bool, default=False, help="bool to determine whether or not to save the trained model")        
     aparser.add_argument("--model_filename", type=str, default="", help="filename for model if it is saved. Should probably start with 'saved_models/' directory")        
     
     args = aparser.parse_args()
 
+    # Define variables that cannot be passed as CL args
+    # FIXME: Figure out a way to pass these as CL arguments
```

Comment:
For rewardFuncs I think we should have a `rewardFuncDirectory` defined in `snakeRewardFuncs.py` which is just a dictionary matching rewardFunc names with the corresponding function, allowing us to pass in the name as a CL argument and use that to grab the function internally. But, I think for this to work smoothly we would need to audit the rewardFunc names to make them more compact and self-similar

Reply:
It looks like Jack did a similar directory idea for the RL algorithms in the main function of `experiments.py` which I like but maybe we could figure out a way to have that directory more generally accessible